Day-II-Calculator/calculator/src/components/ButtonComponents/NumberButton.js

```diff
@@ -1,2 +1,12 @@
 import React from 'react';
 import './Button.css';
+
+function NumberButton(props){
+    return(
+        <div className={props.style}>
+            {props.text}
```

Comment:
I'd love to see some [object destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring) here! For example:
```javascript
function NumberButton({ style, text }){
  return (
    <div className={style}>
      {text}
    </div>
  );
}
```